content/docs/user-guide/experiment-management/hydra-composition.md

```diff
@@ -116,42 +96,27 @@ train:
     momentum: 0.9
 ```
 
-### Building a Pipeline
+## Running experiments
+
+<details>
```

Comment:
Still unsure of this title/ section. We have an entire page with the same name at https://dvc.org/doc/user-guide/experiment-management/running-experiments.

Reply:
Maybe we should try to combine or reorganize these topics (in the future). Rel. https://github.com/iterative/dvc.org/issues/2768 -- feel free to add a check box there.

Reply:
Good idea! Added.